.github/workflows/rebuild-llvm17.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     with:
       version: '17.0'
       full_version: '17.0.6'
-      lto: 'OFF'
+      lto: 'ON'
```

Comment:
I think it makes sense to make `lto` as an input parameter since we need both `lto` and not-`lto` versions for testing.

Reply:
Yes, it is but this is a mostly push triggered job.

Reply:
Let's say we added some new patches and we need to rebuild LLVM. The job will be triggered automatically with LTO=ON. But some tests jobs require LLVM with LTO=OFF. How to trigger rebuild of LLVM with LTO=OFF?

Reply:
I see no solution better than splitting the every LLVM job into two: LTO=ON, LTO=OFF.

Reply:
Looks like it would be the best. I'd use matrix for that
```
    strategy:
      matrix:
        lto: ['ON', 'OFF']
...
lto: ${{ matrix.lto }}
```